### PR TITLE
Awesome Books project

### PR DESCRIPTION
**Pull request by Emmanuelaaron**
### Tools and Techs used
- JavaScript
- HTML 
- Eslint

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    bookContainer.appendChild(card);
+  }
+}
+
+const addBook = () => {
+  const newBook = new Book(title.value, author.value);
+  booksArray.push(newBook);
+  renderBook();
+};
```
Comment (mricanho) [inline js/script.js:43]: - [x] You have a function to add a book, but you need also a function to delete the book, please add it.

- [x] All the data must be preserved in the local storage, please add a function for this(hint: the preserveData function must be called after a book is added and when a book is deleted)

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @Emmanuelaaron @aristides1000, 

It's a good first attempt for this milestone, your add function works as expected and creates an array of objects, goo job there.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] **Remember that all your linters must pass in order to approve all projects, please fix the errors**

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

work hard

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/script.js (hunk 1 above)
Comment (sinansevgi) [reply js/script.js:43]: - [x]  Local storage function works well now it creates elements in the browser storage. However, Still, all the records disappearing when I refresh the page. You should load books from local storage to the page.

Comment (sinansevgi) [reply js/script.js:43]: - [x]  The delete function doesn't work as correctly. You should also consider books with the same title (or I can have multiple copies of the same book). Let's say I have The Double from Fyodor Dostoevsky and The Double from José Saramago, this method will remove both of them when I try to remove one of them.

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @Emmanuelaaron and @aristides1000 , 

Good job so far!
You have delete function now :clap: 
There are some issues that you still need to work on to go further but you are almost there!


 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @Emmanuelaaron and @aristides1000 , 

Local storage works as expected now :+1: 
_Github actions currently don't work. I made the linter checks locally to see if there is an error._ 
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**